pyproject.toml

```diff
@@ -4,6 +4,30 @@ build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
 
+[tool.ruff]
+target-version = "py310"
+line-length = 88
```

Comment:
OOC are all these codes to ignore related to the PR?

Reply:
@eric-czech that's due to the ruff change/commit: [1704d2c](https://github.com/related-sciences/articat/pull/46/commits/1704d2c26d6863d4577bd48a740997822d71c716)